src/sentry/conf/server.py

```diff
@@ -1607,3 +1601,22 @@ def get_sentry_sdk_config():
         'topic': KAFKA_EVENTS,
     },
 }
+
+
+SOUTH_TESTS_MIGRATE = os.environ.get('SOUTH_TESTS_MIGRATE', '0') == '1'
+
+SOUTH_MIGRATION_CONVERSIONS = (
+    # app_name, migration, new_app_name, new_migration_label
+    ('sentry', '0475_auto__add_field_broadcast_cta', 'sentry', '0001_initial'),
```

Comment:
Actually it's probably a good idea to set up several checkpoints here. One for whichever migration was the last for Sentry 10, one for latest master, and do the same for each module.